packages/nextclade-web/src/io/fetchSingleDatasetAuspice.ts

```diff
@@ -0,0 +1,38 @@
+import { isEmpty } from 'lodash'
+import { FatalError } from 'next/dist/lib/fatal-error'
+import { attrStrMaybe, AuspiceTree, Dataset, DatasetFiles } from 'src/types'
+import { removeTrailingSlash } from 'src/io/url'
+import { axiosFetch } from 'src/io/axiosFetch'
+
+export async function fetchSingleDatasetAuspice(datasetJsonUrl_: string) {
+  const datasetJsonUrl = removeTrailingSlash(datasetJsonUrl_)
+
+  const auspiceJson = await axiosFetch<AuspiceTree>(datasetJsonUrl)
+  const pathogen = auspiceJson.meta?.extensions?.nextclade?.pathogen
+
+  if (isEmpty(auspiceJson.root_sequence?.nuc)) {
+    throw new FatalError(`Auspice JSON does not contain required field '.root_sequence.nuc': ${datasetJsonUrl_}`)
+  }
+
+  const currentDataset: Dataset & { auspiceJson?: AuspiceTree } = {
+    path: datasetJsonUrl,
+    capabilities: {
+      primers: false,
+      qc: [],
+    },
+    ...pathogen,
+
+    // HACK: there is no files if dataset comes from Auspice JSON, neither they are needed. What to do?
+    files: {} as unknown as DatasetFiles,
```

Comment:
Threading a separate conversation here (at the same line of code):

I'd remove the "load examples" button from these datasets. As far as I understand there's no way the example data can be fetched (unless, as above, a HTTP address were specified).

Current behaviour from [this URL](https://nextclade-git-feat-ref-and-ann-from-tree-json-nextstrain.vercel.app/?dataset-json-url=https://nextstrain.org/staging/avian-flu/h5n1-cattle-outbreak/genome/nextclade-rn) appears to load an example dataset but when run results in the following error:
```console
Attempted to fetch from an empty URL
```

Reply:
@jameshadfield Should be fixed in [`fc7b8bd` (#1455)](https://github.com/nextstrain/nextclade/pull/1455/commits/fc7b8bd63cf49947c18fcba2cacdb9796c201563)